src/cmdAerospikeIsStable.go

```diff
@@ -188,9 +189,11 @@ func (c *aerospikeIsStableCmd) Execute(args []string) error {
 
 		same := true
 
-		if len(nodes) != len(clusterKeys) {
+		if len(nodes) != len(clusterKeys) { // incorrect number of node cluster keys
 			same = false
-		} else {
+		} else if !c.IgnoreClusterKey && c.NotClusterKey != "" && clusterKeys[0] == c.NotClusterKey { // if cluster key matches the nono list, same=false
```

Comment:
nono?

Reply:
no-no list, ignore this key list, I can update it